pkgs/applications/misc/geoipupdate/default.nix

```
@@ -0,0 +1,21 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
```

common to have a newline here
```suggestion
{ stdenv, buildGoModule, fetchFromGitHub }:

```